### DISCUSSION TITLE
Link information in webhook respone [May 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5819] [Discord Threads]
> 🙋‍♂️ **Rahim Dobani**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### What version are you in? This is available now since   
#### https://github.com/nocodb/nocodb/releases/tag/0.106.0   
####    
#### I would recommend you to switch over to latest release  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### You should now see response something like below   
#### ```   
#### {"type":"records.after.update","id":"7d3fed77-7255-4378-9d42-2b71b1802ec3","data":{"table_id":"md_t5gl2h7absn676","table_name":"country","view_id":"vw_wbkp5tm4fqzmo1","view_name":"country","previous_rows":[{"Id":1,"Title":"india","CreatedAt":"2023-04-28 10:18:07","UpdatedAt":"2023-04-28 10:18:07","cities":[{"Id":1,"Title":"blore"},{"Id":2,"Title":"delhi"}]}],"rows":[{"Id":1,"Title":"indiaa","CreatedAt":"2023-04-28 10:18:07","UpdatedAt":"2023-04-28 10:18:07","cities":[{"Id":1,"Title":"blore"},{"Id":2,"Title":"delhi"}]}]}}   
#### ```  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### "cities": was has-many field in "country" table  
#  
> 🙋‍♂️ **Rahim Dobani**  
#### First Response   
####    
#### {"{\"type\":\"records.after.insert\",\"id\":\"32d97b53-23ff-4eca-ba4c-4b6deeff7477\",\"data\":{\"table_id\":\"md_gd4gcjccpfl5zm\",\"table_name\":\"forms_entry\",\"view_id\":\"vw_031l5gfmiroby7\",\"view_name\":\"nc_xeqt__nc_m2m_1xdj4y6bhi\",\"rows\":[{\"Id\":115,\"CreatedAt\":\"2023-04-28 18:13:07\",\"UpdatedAt\":\"2023-04-28 18:13:07\",\"entry_data\":\"{\\\"employee_name\\\":\\\"ewrewwerewer\\\"}\",\"nc_09n__nc_m2m_gtojdvb2z List\":":[""]}   
####    
#### {"{\"type\":\"records.after.insert\",\"id\":\"b8df53fb-e813-43e1-b76a-bf8e5998b64b\",\"data\":{\"table_id\":\"md_gd4gcjccpfl5zm\",\"table_name\":\"forms_entry\",\"view_id\":\"vw_031l5gfmiroby7\",\"view_name\":\"nc_xeqt__nc_m2m_1xdj4y6bhi\",\"rows\":[{\"Id\":115,\"CreatedAt\":\"2023-04-28 18:13:07\",\"UpdatedAt\":\"2023-04-28 18:13:07\",\"entry_data\":\"{\\\"employee_name\\\":\\\"ewrewwerewer\\\"}\",\"nc_09n__nc_m2m_gtojdvb2z List\":":{"{\"table2_id\":1,\"table1_id\":115}":[{"{\"Id\":1,\"title\":\"Leave Form\"}":[""]}]}}  
#  
> 🙋‍♂️ **Rahim Dobani**  
#### Second Response   
####    
#### {"{\"type\":\"records.after.insert\",\"id\":\"e3461f43-eea6-4e8b-a737-fc9c4e15f7d0\",\"data\":{\"table_id\":\"md_gd4gcjccpfl5zm\",\"table_name\":\"forms_entry\",\"view_id\":\"vw_031l5gfmiroby7\",\"view_name\":\"nc_xeqt__nc_m2m_1xdj4y6bhi\",\"rows\":[{\"Id\":116,\"CreatedAt\":\"2023-04-28 18:13:59\",\"UpdatedAt\":\"2023-04-28 18:13:59\",\"entry_data\":\"{\\\"employee_name\\\":\\\"cvccxcxvcvcx\\\"}\",\"nc_09n__nc_m2m_gtojdvb2z List\":":[""]}   
####    
#### {"{\"type\":\"records.after.insert\",\"id\":\"fd48a307-52aa-4fa3-9e8b-a71fd278cbe1\",\"data\":{\"table_id\":\"md_gd4gcjccpfl5zm\",\"table_name\":\"forms_entry\",\"view_id\":\"vw_031l5gfmiroby7\",\"view_name\":\"nc_xeqt___nc_m2m_1xdj4y6bhi\",\"rows\":[{\"Id\":116,\"CreatedAt\":\"2023-04-28 18:13:59\",\"UpdatedAt\":\"2023-04-28 18:13:59\",\"entry_data\":\"{\\\"employee_name\\\":\\\"cvccxcxvcvcx\\\"}\",\"form\":":{"{\"Id\":1,\"title\":\"Leave Form\"}":{"{\"table2_id\":1,\"table1_id\":116}":[""]}}}  
#  
> 🙋‍♂️ **Rahim Dobani**  
#### I am getting 2 different type of responses from webhook randomly, what could be the issue?  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### can you add more details here about your setup, webhook configurations & scenario?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1101441920628162631)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
